Doc: define PY_SSIZE_T_CLEAN always (#12794)
python/cpython: Doc/c-api/intro.rst

   headers on some systems, you *must* include :file:`Python.h` before any standard
   headers are included.

   It is recommended to define ``PY_SSIZE_T_CLEAN`` always before including

>    ``Python.h``.  See :ref:`arg-parsing` what this macro is.

Reviewer: ```suggestion
   It is recommended to always define ``PY_SSIZE_T_CLEAN`` before including
   ``Python.h``.  See :ref:`arg-parsing` for a description of this macro.
```

>    ``Python.h``.  See :ref:`arg-parsing` what this macro is.

Reviewer: ```suggestion
   ``Python.h``.  See :ref:`arg-parsing` for a description of this macro.
```
